docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+### Genesis Block
+This configuration is used to define the genesis block header. You generally don't need to change these. (except the `gasLimit` field) 
+
+#### `nonce`
+The result of the mining process iteration is this value. It can be any value in the genesis block, which is commonly set to `0x0`.
+
+#### `timestamp`
+The timestamp of the creation of the genesis block.
+
+#### `extraData`
+Optional extra data that can be included in the genesis block. This is commonly set to `0x`.
+
+#### `gasLimit`
+The total amount of gas that can be used in a single block. It should be set to the same value as in [the fee config](#fee-config). The value `0x7a1200` is hexadecimal and it's equal to `8,000,000`.
+
+#### `difficulty`
+The difficulty level applied during the nonce discovering of this block. It is usually set to `0x0` for the genesis block.
+
+#### `mixHash`
+[Quotation from Explanation of genesis file](#resources)  
+> The combination of nonce and `mixHash` must satisfy a mathematical condition described in the Yellowpaper, 4.3.4. Block Header Validity, (44). It allows to verify that the Block has really been cryptographically mined, thus, from this aspect, is valid. 
+
+This isn't so important for the genesis block. You can set it to `0x0000000000000000000000000000000000000000000000000000000000000000`.
+
+#### `coinbase`
+This is the address of the miner who mined the genesis block. It is usually set to `0x0000000000000000000000000000000000000000` for the genesis block.
+
+#### `parentHash`
+This is the Keccak 256-bit hash of the entire parent block’s header. It is usually set to `0x0000000000000000000000000000000000000000000000000000000000000000` for the genesis block.
+
+#### `gasUsed`
+This is the amount of gas used by the genesis block. It is usually set to `0x0`.
+
+#### `number`
+This is the number of the genesis block. It is usually set to `0x0`.
+
+#### `airdropHash` `baseFeePerGas` `airdropAmount`
+You can remove or leave these fields as they are. I'd recommend removing them. I'm not sure what they do.
+
+### Native Token Allocation
+We've done this part of the configuration [using the wizard](#airdropping-native-tokens).
+It's pretty straightforward. You can set the `alloc` to a JSON object with the following properties:
+
+```json
+{
+  "address_without_0x": {
+    "balance": "balance"
+  }
+}
+```
+
+Two important things to notice here:
+- The `address_without_0x` is the address of the account that will receive the tokens. Must **NOT** include the `0x` prefix.
+- Balance is the amount of tokens that will be allocated **in wei** to the account.
+
+In our genesis file. The balance is set to `0xd3c21bcecceda1000000` which corresponds to `1000000000000000000000000` in decimal. You can also type in the balance in decimal. Notice that this number is in wei and it's equal to 1,000,000 Avax. You can use [Snowtrace unitconverter](https://snowtrace.io/unitconverter) to convert between units.
+
+# Creating a Custom Genesis File
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) You shouldn't have multiple `<h1>`s in the same document.

> MD025/single-title/single-h1: Multiple top-level headings in the same document

[More info: MD025](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document)